kadi/events/models.py

```diff
@@ -1466,23 +1481,33 @@ def get_target_attitudes(cls, event, msidset):
 
     @classmethod
     @import_ska
-    def get_one_shot(cls, guide_start, one_shot_msid):
+    def get_one_shot(cls, guide_start, aux_msid):
         """
         Define one_shot attribute (one-shot attitude update following maneuver)
         """
-        # No acq => guide transition so no one-shot defined.  Use 0.0 as a convenience.
+        # No acq => guide transition so no one-shot defined.  Use -99.0 as a convenience.
         if guide_start is None:
             return -99.0
 
-        t0 = DateTime(guide_start).secs
-        n_times = len(one_shot_msid.times)
-        i0, i1 = np.searchsorted(one_shot_msid.times, [t0 - 10, t0 + 10])
-        if i0 == 0 or i1 == 0 or i0 == n_times or i1 == n_times:
-            raise ValueError('Unable to find complete data for one-shot attitude value'
-                             ' {} {} {} {} {}'
-                             .format(i0, i1, t0, np.min(one_shot_msid.times),
-                                     np.max(one_shot_msid.times)))
-        return np.max(one_shot_msid.vals[i0:i1])
+        aoatter1 = aux_msid['aoatter1']
+        aoatter2 = aux_msid['aoatter2']
+        aoatter3 = aux_msid['aoatter3']
+        if len(aoatter2.times) == 0:
```

Comment:
I think that this can never happen because of [this checking](https://github.com/sot/kadi/blob/f1b13436381285dbb7956d27528777dc745638d0/kadi/events/models.py#L1505).  But fair enough to still check.